javatests/arcs/core/host/ParticleRegistrationTest.kt

```diff
@@ -0,0 +1,88 @@
+package arcs.core.host
+
+import arcs.core.sdk.Particle
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class ParticleRegistrationTest {
+    @Test
+    fun serviceLoader_allParticlesAreRegistered() {
+        var foundProdHost = false
+        var foundTestHost = false
+
+        ServiceLoaderHostRegistry.instance.availableArcHosts().forEach { host: ArcHost ->
+            when (host) {
+                is ProdHost -> {
+                    assertThat(TestProdParticle::class).isIn(
+                        host.registeredParticles()
+                    )
+
+                    foundProdHost = true
+                }
+                is TestHost -> {
+                    assertThat(TestHostParticle::class).isIn(
+                        host.registeredParticles()
+                    )
+                    foundTestHost = true
+                }
+            }
+        }
+        assertThat(foundProdHost).isEqualTo(true)
+        assertThat(foundTestHost).isEqualTo(true)
```

Comment:
These could just be `assertThat(...).isTrue()`

Reply:
DOne

Reply:
Not done?